config/status_checks.yml

```diff
@@ -10,6 +10,9 @@ deploys:
   - app: 'identity-saml-sinatra'
     env: 'prod'
     host: 'https://prod-identity-saml-sinatra.app.cloud.gov/'
+  - app: 'identity-pki'
+    env: 'prod'
+    host: 'https://checking-deploy.pivcac.prod.login.gov'
```

Comment:
So we need to have *something* to go in the wildcard domain here, and while I do love `omg-r-u-srsly-expired.pivcac.` that [we use for our cert healthchecks](https://github.com/18F/identity-devops/blob/9c6068c333aa08043ffe708cdf48e9fa667bd14b/terraform/modules/newrelic/pivcac.tf#L7) I figured this was hopefully clear enough